### PR TITLE
add discount audio for maximum great sound

### LINKED ISSUE #12: Play music
Once we have real paths to music files (#11) we need to actually play some music.

### PR DESCRIPTION
**Pull request by ungoldman**
Working audio! Click a file to play! You can even pause and affect the volume!

Caveat: *extremely* hacky.

> resolves #12

### CODE REVIEW

**Comment by ungoldman** [comment]
I'm leaving town tomorrow morning and won't be back until the 25th, so unlikely I'll be able to contribute much more until then. GLHF @bcomnes!

File: renderer/models/player.js [hunk 1]
```diff
@@ -1,9 +1,45 @@
...
+      }, done)
+    },
+    prev: (data, state, send, done) => {
+      console.log('not yet implemented')
+      done()
+    },
+    next: (data, state, send, done) => {
+      console.log('not yet implemented')
+      done()
+    }
```
Comment (ungoldman) [inline renderer/models/player.js]: this whole model is fudged and should be refactored/rewritten soon

**Comment by bcomnes** [comment]
Wow! it freaking works!

**Comment by bcomnes** [comment]
Glorious work @ungoldman

**Pull request merged**